src/runtime/pythonengine.cs

```diff
@@ -520,6 +540,29 @@ internal static PyObject RunString(string code, IntPtr? globals, IntPtr? locals,
                 }
             }
         }
+
+        private static Stream LoadResource(string name)
+        {
+            var args = new ResolveStreamResouceArgs()
+            {
+                Name = name
+            };
+            Stream stream;
+            if (StreamResouceResolve != null)
+            {
+                foreach (Func<ResolveStreamResouceArgs, Stream> resolver in StreamResouceResolve.GetInvocationList())
+                {
+                    stream = resolver(args);
+                    if (stream != null)
+                    {
+                        return stream;
+                    }
+                }
+            }
+            Assembly assembly = Assembly.GetExecutingAssembly();
+            stream = assembly.GetManifestResourceStream("clr.py");
+            return stream;
+        }
```

Comment:
I don't quite understand this, what issue does this resolve? Do you need to override `clr.py`? Why? This should probably be in a separate PR for more discussion.